README.md

```diff
@@ -2,9 +2,13 @@
 Current source code for Launch's custom flight hardware utilizing Zephyr RTOS. This includes but is not limited to the Backplane (modules), GRIM and POTATO.  
 
 ### Compiling
-To compile, you can look at the commands in the Makefile. Type make and the board name. If this is a backplane module, don't include the word module (i.e `make sensor`)
-You can just run `make` with no arguments if you plan on compiling the same board again. If you compile for another baord, 
-a pristine build should occur automatically.
+To compile, navigate to the directory of the application (`app/` folders) and look for a makefile.
```

Comment:
Can technically do it from the root proj dir with the Makefile too, since the top level Makefile includes these. We should maybe state the three different options.

Reply:
I tried doing that and it doesn't work :| i can try to fix the root makefile and update the docs accordingly